### PR TITLE
Fixed issue where pgAgent jobs were not getting dropped from properties tab. #5834

### PR DESCRIPTION
**Pull request by pravesh-sharma**

### CODE REVIEW

File: web/pgadmin/misc/properties/CollectionNodeProperties.jsx [hunk 1]
```diff
@@ -133,7 +133,9 @@ export function CollectionNodeView({
         _type: row.original._type,
       }));
     } else {
-      selRows = selRowModels.map((row) => row.original.oid);
+      let nodeObj = pgBrowser.Nodes[itemNodeData?._type.replace('coll-', '')];
+      let schema = nodeObj.getSchema?.call(nodeObj, treeNodeInfo, itemNodeData);
```
Comment (adityatoshniwal) [inline web/pgadmin/misc/properties/CollectionNodeProperties.jsx:137]: `schema` is already been set at one more place. Use the same schema obj. Do not recreate the logic.

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

File: web/pgadmin/misc/properties/CollectionNodeProperties.jsx (hunk 1 above)
Comment (pravesh-sharma) [reply web/pgadmin/misc/properties/CollectionNodeProperties.jsx:137]: The only place schema is being used is inside the useEffect where table columns are being created, so I reused that logic. I'll check again if we have reference to schema at any other place which can be reused.

Comment (adityatoshniwal) [reply web/pgadmin/misc/properties/CollectionNodeProperties.jsx:137]: The useEffect schema can be re-used. Create a reference to it using `const schemaRef = useRef()` and then `schemaRef.current = <code>`. Then you can use `schemaRef` directly.

Comment (pravesh-sharma) [reply web/pgadmin/misc/properties/CollectionNodeProperties.jsx]: Please review now.

**Review by adityatoshniwal — APPROVED** [review]

File: web/pgadmin/misc/properties/CollectionNodeProperties.jsx [hunk 2]
```diff
@@ -265,7 +266,7 @@ export function CollectionNodeView({
           );
         });
     }
-  }, [itemNodeData, node, item, reload]);
+  }, [itemNodeData, node, item, reload, schemaRef.current]);
```
[the remark this answers is not in the document]
Comment (adityatoshniwal) [reply web/pgadmin/misc/properties/CollectionNodeProperties.jsx]: You can mark this as resolved.

**Pull request merged**